### PR TITLE
Adding useful link

### PR DESCRIPTION
**Pull request by get-itips**
@dariomws What do you think about this addition?
closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2563

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by dariomws** [comment]
@get-itips I think the user is confused about how a cutover migration works. There is no integration between Exchange Online and on-premises. Therefore, after you finish all these steps (cutover migration is complete), you need to configure new Outlook profiles. During the profile configuration wizard, it will connect to Exchange Online autodiscover because you cleaned the AutodiscoverInternalUri, and created a CNAME to point to EXO.

Maybe it's a good idea to add a sentence or note that you need to create new Outlook profiles.

**Comment by get-itips** [comment]
> @get-itips I think the user is confused about how a cutover migration works. There is no integration between Exchange Online and on-premises. Therefore, after you finish all these steps (cutover migration is complete), you need to configure new Outlook profiles. During the profile configuration wizard, it will connect to Exchange Online autodiscover because you cleaned the AutodiscoverInternalUri, and created a CNAME to point to EXO.
> 
> Maybe it's a good idea to add a sentence or note that you need to create new Outlook profiles.

You are right, thanks, can you take a look now?

File: Exchange/ExchangeOnline/mailbox-migration/cutover-migration-to-office-365.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ The main steps you perform for a cutover migration are shown in the following il
 
 8. The administrator completes post-migration tasks in Microsoft 365 or Office 365 (assigns licenses to users and creates an Autodiscover Domain Name System (DNS) record), and optionally decommissions the on-premises Exchange servers.
 
-9. The administrator sends a welcome letter to users to tell them about Microsoft 365 or Office 365 and to describe how to sign in to their new mailboxes.
+9. The administrator sends a welcome letter to users to tell them about Microsoft 365 or Office 365 and to describe how to sign in to their new mailboxes. (See [Create an Outlook profile](https://support.microsoft.com/office/overview-of-outlook-e-mail-profiles-9073a8ac-c3d6-421d-b5b9-fcedff7642fc) and [Overview of Outlook e-mail profile](https://support.microsoft.com/office/overview-of-outlook-e-mail-profiles-9073a8ac-c3d6-421d-b5b9-fcedff7642fc) for information on creating new Outlook profiles).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/cutover-migration-to-office-365.md:84]:
```suggestion
9. The administrator sends a welcome letter to users to tell them about Microsoft 365 or Office 365 and to describe how to sign in to their new mailboxes. See [Create an Outlook profile](https://support.microsoft.com/office/create-an-outlook-profile-f544c1ba-3352-4b3b-be0b-8d42a540459d) and [Overview of Outlook e-mail profiles](https://support.microsoft.com/office/overview-of-outlook-e-mail-profiles-9073a8ac-c3d6-421d-b5b9-fcedff7642fc) for information on creating new Outlook profiles.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by dariomws — APPROVED** [review]
Maybe with the Overview article is enough.

**Comment by scanum** [comment]
@Kateyanne 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**